src/plugins/intel_cpu/src/nodes/executors/implementation_utils.hpp

```diff
@@ -88,4 +93,126 @@ size_t postOpsNumbers(const Config& config) {
     return config.postOps.size();
 }
 
+template <typename Attrs>
+struct RequiredNoFallback {
+    std::optional<executor::Config<Attrs>> operator()(const executor::Config<Attrs>&) const {
+        return {};
+    }
+};
+
+template <typename Attrs>
+struct SupportsAnyConfig {
+    bool operator()(const executor::Config<Attrs>&) const {
+        return true;
+    }
+};
+
+template <typename Attrs>
+struct AcceptsAnyShape {
+    bool operator()(const Attrs&, const PostOps&, const MemoryArgs&) const {
+        return true;
+    }
+};
+
+template <typename Primitive, typename Attrs>
+struct CreateDefault {
+    ExecutorPtr operator()(const Attrs& attrs,
+                           const PostOps& postOps,
+                           const MemoryArgs& memory,
+                           const ExecutorContext::CPtr& context) const {
+        return std::make_shared<Primitive>(attrs, postOps, memory, context);
+    }
+};
+
+template <typename Primitive,
+          typename Attrs,
+          typename ShapeAgnosticData = DnnlShapeAgnosticData,
+          typename Instantiator = DefaultInstantiator<Primitive, Attrs, ShapeAgnosticData>>
+struct CreateDnnlDefault {
+    ExecutorPtr operator()(const Attrs& attrs,
+                           const PostOps& postOps,
+                           const MemoryArgs& memory,
+                           const ExecutorContext::CPtr& context) const {
+        return std::make_shared<DnnlFCExecutor<Primitive, Attrs, DnnlShapeAgnosticData, Instantiator>>(attrs,
```

Comment:
Is it a good idea to always create `DnnlFCExecutor` when we need "DnnlDefault"? If `DnnlFCExecutor` is used as a default Dnnl executor, may be it makes sense to rename it accordingly?